packages/bot-web-ui/src/components/dashboard/tutorial-tab/guide-content/index.tsx

```diff
@@ -73,30 +57,37 @@ const GuideContent = observer(({ guide_list }: TGuideContent) => {
     return React.useMemo(
         () => (
             <div className='tutorials-wrap'>
-                {tour_guides && (
-                    <Text align='center' weight='bold' color='prominent' line_height='s' size={is_mobile ? 'xxs' : 's'}>
-                        <Localize i18n_default_text='Step-by-step guides' />
-                    </Text>
-                )}
-                {tour_guides && (
+                {guide_tab_content?.length !== 0 && (
```

Comment:
Not a major thing but can we just check `guide_tab_content?.length` instead of checking the value of it? If it's undefined or 0 it will be false else it will be true.

Reply:
done ✅